homework.rb

```diff
@@ -1,6 +1,15 @@
-VARIANT = 0
+VARIANT = 1
 
-def generate
-  # put your solution here (delete the line below)
-  rand(1000000).to_s + '-' + rand(1000000).to_s
+def generate 
+
+  def random_12_numbers 
+      return rand(1000..10000)
+  end
+
+
+  first_random_four_numbers = random_12_numbers()
```

Comment:
it's a bad practice in ruby to use brackets in method call if u aren't passing any args